Admin.java

```diff
@@ -0,0 +1,32 @@
+public class Admin {
+
```

Comment:
is separate admin class needed?

Reply:
admin class was deleted and it is mapped to user role